geth/transactions/txqueue_manager.go

```diff
@@ -293,13 +292,13 @@ func (m *Manager) DiscardTransaction(id common.QueuedTxID) error {
 }
 
 // DiscardTransactions discards given multiple transactions from transaction queue
-func (m *Manager) DiscardTransactions(ids []common.QueuedTxID) map[common.QueuedTxID]common.RawDiscardTransactionResult {
-	results := make(map[common.QueuedTxID]common.RawDiscardTransactionResult)
+func (m *Manager) DiscardTransactions(ids []string) map[string]RawDiscardResult {
```

Comment:
We can get rid of this method. There is no `CompleteTransactions`, only `CompleteTransaction`.

Reply:
That's been used on the [backend](https://github.com/status-im/status-go/blob/0e05743109560ce60b8b95ccad80669a84b55be3/geth/api/backend.go#L259)